components/suggest/src/rs.rs

```diff
@@ -301,3 +303,26 @@ pub(crate) struct DownloadedMdnSuggestion {
     pub keywords: Vec<String>,
     pub score: f64,
 }
+
+/// Weather data to ingest from a weather record
+#[derive(Clone, Debug, Deserialize)]
+pub(crate) struct DownloadedWeatherData {
+    pub weather: DownloadedWeatherDataInner,
+}
+#[derive(Clone, Debug, Deserialize)]
+pub(crate) struct DownloadedWeatherDataInner {
+    pub keywords: Vec<String>,
+    // Remote settings doesn't support floats in record JSON so we use a
+    // stringified float instead. If a float can't be parsed, this will be None.
+    #[serde(deserialize_with = "de_stringified_f64")]
```

Comment:
`#[serde(default)]` is required with the custom deserializer. Otherwise, it will error out when `score` is missing from the JSON payload.
```suggestion
    #[serde(default)]
    #[serde(deserialize_with = "de_stringified_f64")]
```